scss/_patterns_notifications.scss

```diff
@@ -187,6 +198,11 @@ $notification-text-margin-bottom: $spv--large - $spv-nudge;
 
     .p-notification__action {
       @extend %default-text;
+      color: $color-link;
```

Comment:
These need to be themed as well. I think we haven't exported them yet to CSS variables, so we will need some kind of `$colors--theme--link-default` and `$colors--theme--link-visited` in the theme, with values based on those variables:

<img width="579" alt="image" src="https://github.com/canonical/vanilla-framework/assets/83575/353df1b1-8528-4203-a3c6-c30d705b11b9">